pkg/plugins/manager/loader/finder/local.go

```diff
@@ -79,14 +80,23 @@ func (l *Local) Find(ctx context.Context, src plugins.PluginSource) ([]*plugins.
 			continue
 		}
 
+		// Skip core plugins if the feature flag is enabled and the plugin is in the skip list.
+		// It could be loaded later as an external plugin.
+		if l.cfg.Features != nil && l.cfg.Features.IsEnabled(featuremgmt.FlagDecoupleCorePlugins) &&
+			src.PluginClass(ctx) == plugins.ClassCore &&
+			l.cfg.SkipCorePlugins[plugin.ID] {
+			l.log.Info("Skipping plugin loading as it is a core plugin", "pluginID", plugin.ID)
```

Comment:
Nit: Probably not a big deal now because this is under a feature flag, but I feel like this should be `Debug` level once the feature flag is removed :)